cmd/manager/main.go

```diff
@@ -71,7 +72,10 @@ func main() {
 
 	// Create a new Cmd to provide shared dependencies and start components
 	log.Info("Setting up manager")
-	mgr, err := manager.New(cfg, manager.Options{MetricsBindAddress: metricsAddr, Port: 9443})
+	// Setting trainedmodel's resync period to 60s
+	var syncPeriod = 60 * time.Second
```

Comment:
we do not need this resync any more if we delete the trained model after inference service is gone right?